build.sbt

```diff
@@ -119,3 +119,22 @@ lazy val example = project.module
       "white-color"     -> "#FFFFFF"
     )
   )
+
+lazy val ztest = project
+  .dependsOn(baseJVM, effectJVM)
+  .in(file("ztest"))
+  .settings(stdSettings("ztest"))
+
+/** A project just for the console.
+  * Applies only the settings necessary for that purpose.
+  */
+lazy val repl = project
+  .dependsOn(ztest % "compile->test")
+  .dependsOn(benchmarks)
+  .settings(
+    console := (console in Test).value,
+    scalacOptions --= Seq("-Yno-imports", "-Ywarn-unused:imports", "-Xfatal-warnings"),
+    initialCommands in console += """
+      |import scalaz._
```

Comment:
just this?

Reply:
Should also import `ztest` and `Scalaz._`.